flutter/lib/src/event_processor/flutter_enricher_event_processor.dart

```diff
@@ -153,6 +153,7 @@ class FlutterEnricherEventProcessor implements EventProcessor {
       // See https://github.com/flutter/flutter/issues/83919
       // 'window_is_visible': _window.viewConfiguration.visible,
       if (renderer != null) 'renderer': renderer,
+      if (_appFlavor != null) 'appFlavor': _appFlavor!,
```

Comment:
I don't think this can be tested without an integration test with a flavor set up. I'm really not sure whether that's worth the effort though.

Reply:
That's fine for me, could also be done separately when someone has the chance